scripts/release.sh

```diff
@@ -12,18 +12,20 @@ fi
 
 MAVEN_LOGIN="${1}"
 MAVEN_PASSWORD="${2}"
-SIGNING_KEY="${3}"
+GPG_SIGNING_KEY="${3}"
```

Comment:
it was an unused variable.
fixing this should change nothing, because this variable was also exported I think :)